core/templates/dev/head/css/oppia.css

```diff
@@ -2144,3 +2150,126 @@ md-card.preview-conversation-skin-supplemental-card {
 .trained-feedback-selection:hover {
   color: #00897B;
 }
+
+.oppia-modal-information-card .modal-dialog {
+  margin: 100px auto;
+  width: 400px;
+}
+
+.oppia-modal-information-card .modal-dialog .modal-content {
+  border-radius: 0;
+}
+
+.oppia-modal-information-card .modal-dialog .modal-content md-card {
+  border-radius:0;
+  margin: 0;
+  padding: 0;
+}
+
+.oppia-information-card-title {
+  bottom: 0;
+  color: white;
+  font-family: "Capriola", "Roboto", Arial, sans-serif;
+  margin: 12px 0;
+  padding: 0;
+  position:absolute;
+}
+
+.oppia-information-card-logo-thumbnail {
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 300px;
+  padding: 20px;
+  position: relative; 
+}
+
+.oppia-information-card-content {
+  padding: 10px 20px 10px 20px;
+}
+
+.oppia-information-card-content p {
+  display: block;
+  font-size: 1.0em;
+  line-height: 1;
+  padding: 12px 0 24px 0;
+  position: relative;
+  word-spacing: 0; 
+}
+
+.oppia-information-card-content .card-metrics {
+  margin-top: 10px;
+  padding: 0;
+}
+
+.oppia-information-card-content .card-metrics li {
+  font-size: 14px;
+  list-style-type: none;
+  text-align: center;
+}
+
+.oppia-information-card-content .card-metrics ul img {
+  height: 24px;
+  margin-right: 2px;
+  width: 24px;
+}
+
+.oppia-contributors-more-circle {
+  background: #888;
+  border-radius: 50%;
+  border-color: black;
+  height: 24px;
+  padding-top: 3px;
+  text-align: center; 
+  width: 24px;
+}
+
+.oppia-information-card-content .card-metrics li .fa {
+  font-size: 24px;
+  margin-right: 5px;
+}
+
+.oppia-social-media-and-tags {
+  margin-top: 20px
+}
+
+.oppia-social-media-and-tags md-chips md-chip {
+  font-size: 12px;
+  line-height: 1em;
+  margin: 0;
+  padding: 0;
+}
+
+.oppia-information-card-tooltip-more {
+  color: blue;
+}
+
+/* This is needed to overide the maximum width of tooltip
+   otherwise  breaks when you have longer text in tooltip */
+.tooltip-inner {
```

Comment:
This is because we don't truncate but we just limit displayed tags to maximum of two tags. So should we truncate if tags are long like that one?. 
That and I am going to remove the inner class .. It looks like appending to the body was just enough. Thanks